File: lib.teacher.base/src/main/java/com/prance/lib/teacher/base/http/Exceptions.java
package com.prance.lib.teacher.base.http;

/**
 * Created by shenbingbing on 16/4/22.
 */
public class Exceptions
{
    public static void illegalArgument(String msg, Object... params)
    {
        throw new IllegalArgumentException(String.format(msg, params));
    }


}

File: lib.teacher.base/src/main/java/com/prance/lib/teacher/base/http/Exceptions.kt
package com.prance.lib.teacher.base.http

/**
 * @author shenbingbing
 * @date 16/4/22
 */
object Exceptions {
    fun illegalArgument(msg: String, vararg params: Any) {
        throw IllegalArgumentException(String.format(msg, *params))
    }
}
